node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3249,6 +3574,26 @@ mod tests {
     #[test]
     fn initial_standard_gossip_does_not_produce_neighborship_if_destination_degree_is_already_full()
     {
+        /*
+        Destination Database ==>
+            N2  N3
+             \  /
+        N1---Dest---Third
```

Comment:
Also confusing: using the name `Third` while you also using numeric indexes for the unimportant Nodes and so the Node means `N3` in the same picture. Instead of `Third` it could be named, for example, `Middle`.